.buildkite/scripts/pipelines/pull_request/pipeline.js

```diff
@@ -104,8 +104,8 @@ const uploadPipeline = (pipelineContent) => {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/osquery_cypress.yml'));
     }
 
-    if (await doAnyChangesMatch([/^x-pack\/plugins\/uptime/])) {
-      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/uptime.yml'));
+    if (await doAnyChangesMatch([/^x-pack\/plugins\/synthetics/])) {
```

Comment:
not sure how this is different from what we have in file task.groovy, but we also need to change this i guess for buildkite